src/actions/transactionBuilder.js

```diff
@@ -29,6 +29,15 @@ export let addOperation = (() => {
   }
 })();
 
+export const DUPLICATE_OPERATION = 'DUPLICATE_OPERATION';
+export function duplicateOperation(opId) {
```

Comment:
Should we name this `opId` as `sourceOpId`? It might be a little bit more clear since we're assigning a different value to `opId`.